test/extensions/filters/http/grpc_stats/config_test.cc

```diff
@@ -20,10 +20,10 @@ namespace {
 class GrpcStatsFilterConfigTest : public testing::Test {
 protected:
   void SetUp() override {
-    std::string json_string = "{}";
-    Json::ObjectSharedPtr json_config = Json::Factory::loadFromString(json_string);
+    envoy::config::filter::http::grpc_stats::v2alpha1::FilterConfig config{};
+    config.set_emit_filter_state(true);
```

Comment:
Please also have tests where you don't emit filter state.

Reply:
Done.